tests/e2e/e2e_setup_test.go

```diff
@@ -929,6 +930,48 @@ func (s *IntegrationTestSuite) writeAddRemoveConsumerProposals(c *chain, consume
 	s.Require().NoError(err)
 }
 
+func (s *IntegrationTestSuite) writeLiquidStakingParamsUpdateProposal(c *chain) {
+	type ParamInfo struct {
+		Subspace string  `json:"subspace"`
+		Key      string  `json:"key"`
+		Value    sdk.Dec `json:"value"`
+	}
+
+	type ParamChangeMessage struct {
+		Title       string      `json:"title"`
+		Description string      `json:"description"`
+		Changes     []ParamInfo `json:"changes"`
+		Deposit     string      `json:"deposit"`
+	}
+
+	paramChangeProposalBody, err := json.MarshalIndent(ParamChangeMessage{
+		Title:       "liquid staking params update",
+		Description: "liquid staking params update",
+		Changes: []ParamInfo{
+			{
+				Subspace: "staking",
+				Key:      "GlobalLiquidStakingCap",
+				Value:    sdk.NewDecWithPrec(30, 2), // 30%
+			},
+			{
+				Subspace: "staking",
+				Key:      "ValidatorLiquidStakingCap",
+				Value:    sdk.NewDecWithPrec(100, 2), // 100%
+			},
+			{
+				Subspace: "staking",
+				Key:      "ValidatorBondFactor",
+				Value:    sdk.NewDec(-1), // -1
```

Comment:
What was the reasoning for these value's? I think it might make sense to use the params that will be on mainnet:
* GlobalLiquidStakingCap: 25%
* ValidatorLiquidStakingCap: 50%
* ValidatorBondFactor: 250

Reply:
Is this 250 or 250%? @sampocs

Reply:
just 250 - it's not a percentage, it's a multiplier

Reply:
Okay